tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+
+When we have bigger projects with different environments, such as React JS projects, we will run tests on our local machines. For now, we can focus on reading tests and building our skills to pass them.
+
+## Reading Tests Without Learning Syntax
+
+Leila is jumping into a JavaScript project. They're pairing with another developer, Nadia, who has been working on implementing FizzBuzz.
+
+Nadia has already set up the tests for FizzBuzz. Leila wants to jump into coding, but they need to take a moment to read through the tests.
+
+What can Leila determine from reading tests in a syntax they don't know? Let's read through one test and make predictions.
+
+This is in a file named `test/fizzbuzz.test.js`
+
+<!-- prettier-ignore-start -->
+```js
+describe('fizzBuzz', () => {
+  test('returns Fizz on multiples of 3', () => {
+    // Arrange
+    const num = 9;
+
+    // Act
+    const result = fizzBuzz(num);
+
+    // Assert
+    expect(result).toEqual('Fizz');
+  });
+});
+```
+<!-- prettier-ignore-end -->
+
+### The File Itself
+
+First, Leila can quickly orient themself with the file name and where it's located. It seems like there is a `test` directory, and a file named `fizzbuzz.test.js` in it.
+
+From this evidence, they can at least hypothesize this:
+
+- This file is dedicated to validating the correctness of the `fizzBuzz` function
+- It's unlikely that the `fizzBuzz` implementation is in this file, or even this folder
+
+### !callout-info
+
+## Confidently Make Hypotheses
+
+As Leila works through this, they will need to make a lot of assumptions. Leila can take the time to fact-check all of her assumptions, which is a completely valid thing to do (and typically a good strategy for life). However, in this situation, Leila can _also_ confidently rely on their skills, intuition, context, and help from others to make theories about what's going on.
+
+<br/>
+
+"Maybe the `test` folder only holds test files, and we have a `.test.js` file for each function." Making theories like this will help Leila move forward in their learning. It will also help them correct themself and their working knowledge more easily!
+
+### !end-callout
+
+### Hypothesizing About `describe`
+
+Let's look at block of code that begins on the first line, and closes on the last line.
+
+<!-- prettier-ignore-start -->
+```js
+describe('fizzBuzz', () => {
+  // ...
+});
+```
+<!-- prettier-ignore-end -->
+
+Even though we don't know what `describe` means here, we can see that its syntax looks similar to a function call with two arguments:
+
+<!-- prettier-ignore-start -->
+```js
+describe( ..., ...
+);
+```
+<!-- prettier-ignore-end -->
+
+We can also see that the first argument is the string literal `'fizzBuzz'`. When Leila sees this, Leila questions, "What is this string literal used for?"
+
+When scanning through the rest of this code, we don't see this string literal being used anywhere obviously, so they will pause on that question for now.
```

Comment:
keep the focus on Leila through this paragraph
```suggestion
After scanning through the rest of the test code, they don't see any obvious place where this string literal is being used, so they set that question aside for the moment.
```

Reply:
+1 to Ansel's rewording here :)